src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The 'hidden' property should not be part of the Animal class constructor as not all animals may have this property. It's specific to Herbivores and should be defined within the Herbivore class.